Source/ApiTemplate/Source/ApiTemplate/CustomServiceCollectionExtensions.cs

```diff
@@ -240,7 +240,7 @@
                                         activity.AddTag(OpenTelemetryAttributeName.Http.RequestContentType, request.ContentType);
 
                                         var user = context.User;
-                                        if (user.Identity.Name is not null)
+                                        if (user.Identity?.Name is not null)
                                         {
                                             activity.AddTag(OpenTelemetryAttributeName.EndUser.Id, user.Identity.Name);
```

Comment:
We should probably assume these are available, which will throw if they are not for some reason.